package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.53.0",
```

Comment:
do we need to specify the usage of this anywhere? The docs seem to mention something about setting this as the parser? do you know if that is required?